### ISSUE TITLE
Sympa Accessibility Issue - 26 - No p element in scope but a p end tag seen

### ISSUE THREAD

**Issue by rm1-uiuc** [sympa-community/sympa#1778]
The University of Illinois at Urbana-Champaign, Technology Services Accessibility Testing team conducted an Accessibility Review of Sympa on September 25, 2023 and found 26 issues that need to be fixed to make the interface accessible for visual impaired users.

Version
-------
6.2.72

Installation method
-------------------
Source Package

Expected behavior
-----------------
if a p end tag is present then p element should be in scope for an accessibility screen reader to properly read it.

Actual behavior
---------------
No p element in scope but a p end tag seen when running automation tool evaluation tool.

Steps to reproduce
------------------
Sympa Accessibility Issue - 26 - was determined by an Automation Tool Evaluation using AInspector and Siteimprove Accessibility Checkers as automated accessibility evaluation tools.

After running these automation tools the following issue was identified:

No p element in scope but a p end tag seen.

HTML Code Validation

The purpose of code review is to make sure that the HTML follows the Document Type Declaration (DTD), which sets out the rules and grammar for the flavor of markup, thus enabling the browser to render the content accordingly. The HTML5 validator [NuValidator](https://validator.w3.org/nu/) was used in this evaluation. Code validation should be done to better conform with HTML standards and thus accessible coding practices. Although several of the errors uncovered by the Validator are minor, compliance means the site will operate as expected when used by assistive technology (AT) (WCAG 2.0 [4.1.1 Parsing](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html)). Often, fixing one or two of those errors early on can resolve many of the errors flagged, since it has a waterfall effect.

Additional information
----------------------
Attached is the Accessibility Review conducted on Sept 25,2023
[Tech Services Review - Accessibility Evaluation Report for Sympa.pdf](https://github.com/sympa-community/sympa/files/13562859/Tech.Services.Review.-.Accessibility.Evaluation.Report.for.Sympa.pdf)

**Comment by ikedas** [comment]
Hi @rm1-uiuc,

Could you please show where the end tag of `p` found, with screenshot or anything else?

**Comment by ikedas** [comment]
If there are no further information, this issue will be closed.
